tests/gis_tests/geos_tests/test_geos.py

```diff
@@ -883,15 +883,15 @@ def test_threed(self):
         # Testing a 3D Point
         pnt = Point(2, 3, 8)
         self.assertEqual((2., 3., 8.), pnt.coords)
-        with self.assertRaises(TypeError):
+        with self.assertRaises(ValueError):
```

Comment:
Assuming the original exception was "'Dimension of value does not match.", that seems a bit friendlier (although it might need a minor language tweak anyway).